rest-server/server/src/main/resources/logback.xml

```diff
@@ -3,7 +3,7 @@
 
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%green(%d{HH:mm:ss.SSS}) %highlight(%-5level) %green(%-17logger{0}) - %msg%n</pattern>
+            <pattern>%d{HH:mm:ss.SSS} %-5level %-17logger{0} - %msg%n</pattern>
```

Comment:
Does this remove coloring from when viewing in terminal to tailing? I know the escape code junk up the logs but we always search or grep them anyway.

Reply:
yes, it removes "colored" logs